src/chrome/content/rules/University-of-Exeter.xml

```diff
@@ -26,6 +30,10 @@
 	<securecookie host="^(?:.*\.)?exeter\.ac\.uk$" name=".+" />
 
 
+	<!-- Cert only matches empslocal.ex.ac.uk -->
```

Comment:
Ah. Makes more sense. Maybe expand the comment slightly?

With regards to my comment about the newton.ex.ac.uk below... I wonder if that can be removed, and noted in the observed subdomain